### DISCUSSION TITLE
Dağıtım şekli

### DISCUSSION THREAD

**Discussion by kublaycelik** [IBAX-io/go-ibax#152] [Rewards]
Bu işi büyük ölçekte yapan fenomenler çok rahat şekilde takipçileri ile paylaşıp daha yüksek oranda airdrop alabilecekler. Halk yerine yine parası olan kazanacaktır. Keşke daha farklı bir sistem ile dağıtılsa hediyeler.

**Comment by byblckd** [comment]
Artık projeler bu şekilde ilerliyor sebebi sosyal medyanın gücü ücretsiz bir şekilde en yüksek katılımı bu şekilde elde ediyorlar. Dağıtılacak miktar deve de kulak misali bu yüzden pek sorun olacağını düşünmüyorum

**Comment by semihtatliel** [comment]
Evet Fenomen sayfaları yöneten kişiler daha fazla kazanacak fakat şöyle de bir durum var eğer o fenomenler olmasa bu gibi projelerin ismi dahi duyulmaz sosyal medya gücünü kullanması gerekiyor projelerin ve bunuda gayet güzel yapıyorlar o yüzden fenomenler hak ediyor diye düşünüyorum kendi şahsi fikrim

**Comment by Grekny** [comment]
şuan inanılmaz hypelı bir coin değil ben tek tük az aboneli youtube kanallarında gördüm.O yüzden bizim gibi girip tek tük ref kasanlarda kazanabilicek gibi duruyor.

**Reply by kublaycelik** [reply]
Telegramda paylaşmayan sayfa kalmadi. Size denk gelmemiş

**Comment by kublaycelik** [comment]
Ben projeyi bir fenomenden duymadım. Kendi araştırmalarım sonucu karşıma çıktı. Araştırarak bulunabiliyor. Sonuçta en çok davet eden 10 kişiye ekstra ödül var. Bayada iyi hani ödüller. Geçen sene avax dan milletin milyon kazandığını düşünürsek, keşke bizler kazanabilmek o parayı demek istemiştim ben.
